homeassistant/components/tractive/config_flow.py

```diff
@@ -69,6 +75,36 @@ async def async_step_user(
             step_id="user", data_schema=STEP_USER_DATA_SCHEMA, errors=errors
         )
 
+    async def async_step_reauth(self, user_input: dict = None) -> FlowResult:
+        """Handle configuration by re-auth."""
+        errors = {}
```

Comment:
I do not fully understand how this is working. I tried to test it as well as possible.

Reply:
Add a confirm step that we call here. The confirm step will show the form and do the validation.

Reply:
I removed the part with reauth, since I do not know how it works and it is hard to test.

Reply:
It's described here <https://developers.home-assistant.io/docs/config_entries_config_flow_handler/#reauthentication> I used this guide recently and it contained all I needed.

Reply:
Why is async_step_reauth_confirm needed?
Why can not that code be inside async_step_reauth?

Reply:
https://github.com/home-assistant/core/pull/54226#discussion_r684721522

Reply:
Like this?
https://github.com/home-assistant/core/pull/54532/commits/919d3cc3465b4b2831d69c34b0c0c3b4ed0ad027